src/main/java/net/honux/chess/attribute/Pointer.java

```diff
@@ -2,30 +2,38 @@
 
 public class Pointer {
 
+    private static final String separator = ",";
     private char x;
     private int y;
-    private final String separator = ",";
 
     public Pointer(char x, int y) {
         this.x = x;
         this.y = y;
     }
 
-    public void setPointer(Pointer pointer){
-        String pointer1 = pointer.toString();
-        this.x = pointer1.split(separator)[0].charAt(0);
-        this.y = Integer.parseInt(pointer1.split(separator)[1]);
+    public void setPointer(Pointer pointer) {
+        this.x = pointer.getX();
+        this.y = pointer.getY();
```

Comment:
```suggestion
        this.x = pointer.x;
        this.y = pointer.y;
```

이렇게도 가능합니다. 한 번 해보세요~
Java에서는 같은 타입의 속성을 참조할 수 있습니다.

Reply:
아 그렇군요. 
이런 생각을 못해봤는데 감사해요 Dion ! :)